conans/test/integration/hook_test.py

```diff
@@ -203,7 +203,7 @@ def complete_hook_test(self):
         self._check_export_pkg(conanfile_cache_path, client.out)
 
         client.run("remove * --force")
-        client.run("export-pkg . -pf . danimtb/testing")
+        client.run('export-pkg . danimtb/testing -pf . ')
```

Comment:
"breaking" change. With the changes introduced (reference optional) now it gets wrong the args in that case.

Reply:
This is also solved with ``-pf=.``. It is a general problem of the way ``argparse`` prints the order in the ``--help`` documentation. 
As this is also broken for other commands and arguments, I wouldn't consider it really breaking.